roadmap.md

```diff
@@ -0,0 +1,53 @@
+# Roadmap
+
+This file provides an overview of the direction this project is heading. The roadmap is organized in steps that focus on a specific theme, for instance, UX.
+
+## Q1 2024
+
+In this phase of the project, the focus is on ad hoc scanners. The expected features are:
+
+**Adhoc Scan​**
+
+It enables users to target particular repositories or entire organizations for scanning purposes. Additionally, users can filter out archived or forked repositories to narrow down the scope of the scan. 
+
+- Targeted repo scan​
+- Targeted org scan​
+- Filter archived repos​
+- Filter forked repos​
+- Custom keyword search​
+
+## Q2 2024
+
+In this phase of the project, the focus is on File Scanner. 
+The expected features are:
+
+
+**File scanner​**
+
+The file scanner helps users understand the content and structure of their files or directories by scanning them thoroughly.
+- Scan directory​
+- Scan individual files
+
+
+
+## Q3 2024
+
+In this phase of the project, the focus is on utilising ML to improve the product. 
+The expected features are:
+
+
+**Bert Model ​**
+
+Using BERT for training and building models enhances language understanding in tasks like question answering. Integrating BERT into scanners improves accuracy by leveraging its contextual understanding of language, enabling more precise analysis of text or code.
+
+- Training and  building models using Bert​
+- Integrate Bert to scanners​
+
+
+
+## Q4 2024 and beyond
+​
+**Pre-commit Hook​**
+
+The pre-commit hook is a multi-language package manager that automatically installs and runs various code linters and checks before each commit, streamlining code review by identifying issues like missing semicolons or trailing whitespace.
```

Comment:
Context here is not right.  Focus on this hook is capability to block secrets getting pushed.